setup.py

```diff
@@ -69,8 +69,8 @@
     lightning_dep = f"pennylane-lightning>={lq_min_release}"
 
 requirements = [
-    pennylane_dep,
-    lightning_dep,
+    "pennylane>=0.36",
+    "pennylane-lightning>=0.36",
```

Comment:
Please update the relevant variables for this above :)

Reply:
https://github.com/PennyLaneAI/catalyst/pull/715/commits/6ac36e3072e862df4a870145db0e5fa18e7d0802